runtime/gc_base/StringTable.cpp

```diff
@@ -20,6 +20,8 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
  *******************************************************************************/
 
+#include "StringTable.hpp"
+
```

Comment:
Would you please remove includs mentioned in .hpp already ("j9protos.h" and "ModronAssertions.h") ?